### PR TITLE
[NetAppFiles] Generate SDK to api-version 2023-05-01

### PR DESCRIPTION
**Pull request by audunn**
## Mandatory Checklist

- Please choose the target release of Azure PowerShell
  - [x] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] N/A

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Comment by dolauli** [comment]
@audunn Please resolve the conflicts.

**Comment by dolauli** [comment]
@audunn There are some issues detected in the `Analyze`. Please resolve them.

File: src/NetAppFiles/NetAppFiles/ChangeLog.md [hunk 1]
```diff
@@ -18,6 +18,13 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Fixed some minor issues
+* Updated to api-version 2023-05-01
+* Added `EncryptionKeySource`, `KeyVaultKeyName`, `KeyVaultResourceId`, `KeyVaultUri`, `IdentityType`, `UserAssignedIdentity`  to `New-AzNetAppFilesAccount` and `Update-AzNetAppFilesAccount`
+* Added new cmdLets `Get-AzNetAppFilesNetworkSiblingSet` and `Update-AzNetAppFilesNetworkSiblingSet` to query and update update the network features of a network sibling set
```
Comment (dolauli) [inline src/NetAppFiles/NetAppFiles/ChangeLog.md:24]: `update update` - one update is redundant.

Comment (dolauli) [reply src/NetAppFiles/NetAppFiles/ChangeLog.md:24]: line#24 and line#25 are the same. Please remove one

**Review by dolauli — CHANGES_REQUESTED** [review]

**Review by dolauli — APPROVED** [review]

**Comment by audunn** [comment]
Note we are be adding a fix for NetApp Account Identity in https://github.com/Azure/azure-powershell/pull/23808

**Pull request merged**